user_guide_src/source/libraries/content_negotiation.rst

```diff
@@ -39,15 +39,15 @@ Media
 =====
 
 The first aspect to look at is handling 'media' negotiations. These are provided by the ``Accept`` header and
-is one of the most complex headers available. A common example is the client telling the server what format it
+are one of the most complex headers available. A common example is the client telling the server what format it
```

Comment:
`is` is referring to the singular usage of the Accept header here, so `is` would be correct. We mention headers (plural) after that, but the subject here is the singular header. 

If you will change this back to `is`, the rest looks good.